### PR TITLE
refs #40 - implement infinite scrolling for /games list of games

### PR DESCRIPTION
**Pull request by nazar**
Hey @mikelax 

I've implemented simple offset based paging for now as I was struggling to figure out how to implement cursor based paging on the server. We can revisit this once more features are out of the way?

### CODE REVIEW

**Comment by nazar** [comment]
refs #40

File: client/src/components/Games/queries.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import gql from 'graphql-tag';
 
 export const gamesQuery = gql`
-  query games($page: Int, $searchOptions: SearchOptions) {
-    games(page: $page, searchOptions: $searchOptions) {
+  query games($offset: Int, $searchOptions: SearchOptions) {
```
Comment (mikelax) [inline client/src/components/Games/queries.js:4]: @nazar  is this offset using a cursor or a true offset? 
shouldn't the cursor approach send the "id" of the last id from the previous request?

Comment (nazar) [reply client/src/components/Games/queries.js:4]: I'm not using cursors for this initial implementation as I was not able to find examples, specifically for how the server handles cursors. The example I've seen mention cursors on the client but not how there are generated on the server and sent to the client, specifically in relation to any existing search parameters.

I didn't want to spend too much time on this. We can revisit once this becomes a performance issue and we have implemented more features.

**Comment by nazar** [comment]
Hey @mikelax - can I go ahead and merge this PR? I'd like to access the features in this branch for #31

**Comment by mikelax** [comment]
@nazar sure. I will do it now.

**Pull request merged**